### PR TITLE
harmonize metric naming convention to ucsm

### PR DESCRIPTION
**Pull request by BerndKue**

### CODE REVIEW

**Review by mithunmgore — APPROVED** [review]
Need to change alert definition for this collector. I will create a PR soon.

File: collectors/UcsServerLicenseCollector.py [hunk 1]
```diff
@@ -14,13 +14,13 @@ def __init__(self, creds, config):
         }
 
     def describe(self):
-        yield GaugeMetricFamily("ucs_server_license", "licenses")
+        yield GaugeMetricFamily("ucsm_server_license", "licenses")
```
Comment (viennaa) [inline collectors/UcsServerLicenseCollector.py:17]: @mithunmgore why would it be renamed to `ucsm`? from the name I'd expect `ucs` only.

**Comment by viennaa** [comment]
@BerndKue we are having a conflict now.

**Comment by BerndKue** [comment]
Well, in that case, the others should be renamed. I actually don't care if the are called ucs_* or ucsm_*.
They just need to have the same naming convention.

**Comment by viennaa** [comment]
@mithunmgore your call, how to name them. Just make sure, they are following the same scheme.

**Comment by mithunmgore** [comment]
@BerndKue @viennaa My idea was to have ucs* for the metrics which are common to ucsc server / ucsm server and ucsm*/ucsc* to the ucsm/ucsc server specific. Ex. UCSC metrics like service profiles/templates will have ucsc* and UCSM metrics like firmware_version, CRC, DIMM will have ucsm*. 
so server license I thought it will be common but now I think I will use ucsm and ucsc.

**Comment by BerndKue** [comment]
I would not make it too complicated:
1. important I think is that people find the metrics coming from the same exporter in Prometheus
2. different names of the metrics make it harder to define alerts. For my part, i would also wonder if there is a typo because one alert says ucs_* and the other ucsm_* or ucsc_*
3. ucs-c rack servers are lately bought with no FI and hence no ucsm. We are already monitoring those servers via IPMI and Redfish exporters.
4. We don't buy any UCSM based compute POD anymore since a while. With in about 3 years we might be without any FI.

So, my recommendation: keep it simple. ;)

**Pull request closed without merging**